### ISSUE TITLE
Cannot construct languages with empty lists.

### ISSUE THREAD

**Issue by LeifAndersen** [nanopass/nanopass-framework-racket#9]
Let's say I have the following language:

```
(define-language Llist
  (terminals
   (symbol (s)))
  (List (l)
        s
        (l ...)))
```

I can use `with-output-language` to make an empty list, and I can use `with-output-language` to also create lists with non-empty sub-lists.

However, when I try to use `with-output-language` to create a list with an empty sublist, I get the following error:

```
(with-output-language (Llist List)
  `(a ()))
```

```
. . with-output-language: expected List but received () in field l of (l ...) from expression (#<syntax:11:4 (quote a)> #<syntax:11:6 (quote ())>)
```

**Comment by LeifAndersen** [comment]
As a temporary workaround, you can do:

```
(with-output-language (Llist List)
  `(a (,'() ...)))
```

But this is obviously very cludgy.

**Comment by LeifAndersen** [comment]
Fixed: 654a311f63c1dc2e1dab1f03762ec0f267f6c801
